src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3376,15 +3429,15 @@ private static bool HasPointerToPointerConversion(TypeSymbol source, TypeSymbol
             Debug.Assert((object)source != null);
             Debug.Assert((object)destination != null);
 
-            return source is PointerTypeSymbol && destination is PointerTypeSymbol;
+            return source.IsPointerOrFunctionPointer() && destination.IsPointerOrFunctionPointer();
```

Comment:
>source.IsPointerOrFunctionPointer() && destination.IsPointerOrFunctionPointer(); [](start = 19, length = 80)

It doesn't look like specification mentions any explicit conversions when function pointers are involved.

Reply:
Function pointers are a type of pointer type. The existing pointer type conversions are covered here: https://github.com/dotnet/csharplang/blob/master/spec/unsafe-code.md#pointer-conversions